src/block/snake.scss

```diff
@@ -0,0 +1,22 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($block-width, 1fr));
+
+  &__block {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    min-width: $block-width;
+    min-height: $block-height;
```

Comment:
In your case the minimum width of the cell is set by grid-template-columns property on line 3. No need for specifying the width here.